engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -623,6 +631,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
               self, jobKey, data, factory, initializationData.get(backendName), restarting, serviceRegistryActor,
               jobStoreActor, callCacheReadActor, jobTokenDispenserActor, backendSingleton, backendName, workflowDescriptor.callCachingMode)
             val ejeaRef = context.actorOf(ejeaProps, ejeaName)
+            context watch ejeaRef
             pushNewJobMetadata(jobKey, backendName)
```

Comment:
why not add the `ejeaRef` to the data here instead of when receiving the `JobStarting` message ?